### PR TITLE
parse password from DSN

### PR DESCRIPTION
**Pull request by skinnyjames**
Apologies, I wasn't able to run a successful test as the tests could not find a config file in LoadConfig - but it works on my end and should be pretty straightforward and unobtrusive.

Could you run your tests against it perhaps?

### CODE REVIEW

**Comment by skinnyjames** [comment]
yikes.  I'll cancel this.

**Comment by rnubel** [comment]
Thanks for the PR! It doesn't look like the failures are your fault; after some investigation, I wrote up https://github.com/rnubel/pgmgr/pull/34 to skip sending empty connection values to the adapter. Possibly a PG version change altered the behavior (it's been ~4 months since the last CI build on master, at which point I was probably on <= 9.4 locally).

If CI passes on my PR, I'll merge it into master and then you can rebase your branch.

File: pgmgr/config.go [hunk 1]
```diff
@@ -160,11 +160,21 @@ func (config *Config) overrideFromURL() {
 	r := regexp.MustCompile("^postgres://(.*)@(.*):([0-9]+)/([^?]+)")
 	m := r.FindStringSubmatch(config.URL)
 	if len(m) > 0 {
-		config.Username = m[1]
+		user := m[1]
 		config.Host = m[2]
 		config.Port, _ = strconv.Atoi(m[3])
 		config.Database = m[4]
 
+		ur := regexp.MustCompile("^(.*):(.*)$")
```
Comment (rnubel) [inline pgmgr/config.go]: Can you use longer variable names here? E.g. `userRegex`, `userMatch`.

**Comment by rnubel** [comment]
Update: #34 passed, and locally, this PR passes after being rebased. So rebase, fix those two variable names, and we're good to go.

File: pgmgr/config.go (hunk 1 above)
Comment (skinnyjames) [inline pgmgr/config.go]: Sure thing!

**Comment by rnubel** [comment]
Merged. Thanks for the contribution!

**Pull request merged**